docs/pages/admin-guides/management/guides/migrating-iam-ic-from-okta-to-teleport.mdx

```diff
@@ -0,0 +1,334 @@
+
+![Migration End Point](../../../../img/identity-center/ic-migrate-end.png)
+
+## Prerequisites
+
+ - A working Teleport cluster
+ - An AWS role configured as per the [Identity Center guide](./aws-iam-identity-center.mdx)
+   for the integration to use
+ - AWS credentials configured for the Teleport auth process to pick up and use 
+   (e.g. as environment vars, system profiles, etc)
+ - An Okta API token, with the following privileges:
+   - View users and their details
+   - View groups and their details
+   - View applications and their details
+ - The AWS Identity Center ARN, AWS region, SCIM base address and SCIM bearer token
+
+## Step 1/7. Install Okta SAML connector
+
+Install Okta SAML connector into Teleport as per the Teleport 
+[Okta as an SSO provider](../../access-controls/sso/okta.mdx) guide.
+
+<Admonition type="note">
+ For the integration to function properly, both AWS Identity Center and Teleport must view the
+ same user set which can be achieved by using the same Okta SAML application for both Identity
+ Center and Teleport SAML connector.
+
+ If this is not possible, the most flexible approach we've found is having an
+ an Okta Group for those users that should have IC access, and then assigning
+ both the Okta Identity Center App and the Okta SAML App used for Teleport SAML
+ connector to that group. This ensures that the same user set is visible across
+ both applications.
+</Admonition>
+
+You will need the Teleport SAML Connector Name and Okta SAML App ID in the next
+step.
+
+## Step 2/7. Install the Teleport Okta integration
+
+We will be using a very limited subset of the Teleport Okta integration in this 
+deployment, disabling all features except periodic user synchronization. This
+configuration is not currently supported by the normal installation UI, so we
+will have to use `tctl` to install it:
+
+```console
+$ tctl plugins install okta \
+    --org ${OKTA_ORG_URL} \
+    --saml-connector ${TELEPORT_SAML_CONNECTOR_NAME} \
+    --app-id ${OKTA_SAML_APP_ID} \
+    --api-token ${OKTA_API_TOKEN} \
+    --no-scim \
+    --no-accesslist-sync \
+    --no-appgroup-sync
+```
+
+This will install the Okta integration and start the User sync service. You can
+monitor the state of the Okta integration in the Teleport Integrations UI.
+
+## Step 3/7. Wait for user sync
+
+To make sure everything is working, wait until the first Okta to Teleport user
+sync has occurred. You can verify this by either
+ - refreshing the user page and finding your Okta users, or
+ - checking the Okta integration status page
+
+Once your Okta users are imported into Teleport, you can progress to the next
+step.
+
+## Step 4/7. Install the Teleport Identity Center Integration.
+
+Again, we need to install the plugin using `tctl`. 
+
+```console
+$ tctl plugins install awsic \
+    --arn ${IDENTITY_CENTER_INSTANCE_ARN} \
+    --region ${IDENTITY_CENTER_INSTANCE_REGION} \
+    --use-system-credentials \
```

Comment:
Can we add more details on use-system-credent? 
Like provide hint that it refers to AWS credentials. Specifically, it should clarify that use-system-credent will allow to load AWS credentials from the local environment in Teleport Auth to authenticate with the AWS IC API.